src/inc/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ target_link_libraries(base_link INTERFACE Threads::Threads ${CMAKE_DL_LIBS})
 if (ATOMIC)
     target_link_libraries(base_link INTERFACE ${ATOMIC})
 endif()
+target_link_libraries(base_link INTERFACE "/usr/lib/x86_64-linux-gnu/libhwloc.so")
```

Comment:
this will break arm/arm64 build

Reply:
arm?
not windows from path point of view?